src/App.js

```diff
@@ -1,25 +1,101 @@
-import logo from './logo.svg';
+// Components
```

Comment:
You organized your code very well! The comments segregating the imports, variables, and what you were trying to accomplish per function makes it easy to follow along.